xwalk_android_tests.gypi

```diff
@@ -194,7 +194,6 @@
           ],
         },
       ],
```

Comment:
@rakuco for example this include is redundant as _xwalk_core_test_apk_ depends on _xwalk_core_shell_apk_java_ which in turn depends on _xwalk_core_shell_apk_ and _xwalk_core_shell_apk_ includes _java_apk.gypi_